src/main/java/se/sundsvall/messaging/api/StatusAndHistoryResource.java

```diff
@@ -322,14 +321,51 @@ ResponseEntity<Statistics> getStats(
 
         return ResponseEntity.ok(stats);
     }
+
+    @Operation(
+        summary = "Get letter delivery statistics by department",
+        responses = {
+            @ApiResponse(
+                responseCode = "200",
+                description = "Successful Operation",
+                useReturnTypeSchema = true
+            ),
+            @ApiResponse(
+                responseCode = "500",
+                description = "Internal Server Error",
+                content = @Content(schema = @Schema(implementation = Problem.class))
+            )
+        }
+    )
+    @GetMapping(
+        value = {"/statistics/departments","/statistics/departments/{department}"},
+        produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE }
+    )
+    ResponseEntity<DepartmentStatistics> getDepartmentStats(
```

Comment:
This will allow empty string as value for department, from and to, which might not be desirable? Maybe add validation that checks that value is null or not empty (not sure if such annotation exists or if it needs to be created, but maybe regex-annotation can be used if it considers null as valid).